Packs/QRadar/ReleaseNotes/2_2_0.md

```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+##### IBM QRadar v3
+- Improved mirroring performance.
+- Added the number of fetched events in the offense context.
+
+#### Mappers
+##### QRadar - Generic Incoming Mapper
+- Fixed the mapping of the number of fetched events.
+
+#### Scripts
+##### QRadarPrintEvents
+- Fixed an issue where there are too many events. Now the print is limited to 20 events.
```

Comment:
Is this the final number?
```suggestion
- The widget now shows just the first 10 events. Previously the widget would fail to load when too many events were fetched.
```